docs/installing_hypershift_cli.md

```diff
@@ -6,23 +6,24 @@ The Hosted Control Plane command-line interface (CLI) is an `oc` command-line [p
 
 ## Installing the Hosted Control Plane CLI
 
-**NOTE: This document is unofficial will be updated later. For now download the CLI binary from this page
+1. [Enable the Hypershift feature](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/provision_hosted_cluster_on_mce_local_cluster.md#configuring-the-hosting-service-cluster)
```

Comment:
We use the term Hypershift and then talk about HCP below.
`Enable the Hypershift hosted control plane feature`